### DISCUSSION TITLE
LDAP auth subtree search

### DISCUSSION THREAD

**Discussion by griznog** [requarks/wiki#6788] [Help / Questions]
Hi, 

I've gotten LDAP/AD auth to work by specifying a specific OU as the base, but unfortunately our users are spread out over many OUs which are beyond my control. How can I specify a search base and then have it do a subtree search for users in all OUs under that base?

Edit: I see the Search scope option for group searches, which is what I want to also set to `sub` for users, but don't see a way to set the scope for the user search. 

Thanks,

griznog

**Comment by anthonyballetta** [comment]
I am also unable to login as a user underneath the parent OU.

EDIT: Figured it out. Remove the OU from the search base.
